message PBMsgReqHeroHouseList {
  required int64 cid = 1;
}
